acceptance-tests/src/test/java/org/eclipse/collections/impl/concurrent/MultiReaderUnifiedSetAcceptanceTest.java

```diff
@@ -303,7 +303,8 @@ private static void runUnifiedSetRetainAllFromList(int shift)
         Assert.assertTrue(set.retainAll(toRetain));
         Assert.assertTrue(set.containsAll(toRetain));
 
-        Assert.assertFalse(set.retainAll(toRetain)); // a second call should not modify the set
+        // a second call should not modify the set
```

Comment:
It feels like all these comments should end with `:`, or they don't read quite the same.

Reply:
I tried to address both readability concerns by adding newlines where they were missing so it's even more obvious that the comment applies to the next statement or expression. I'm open to adding `:` too. And I'm still open to the idea that this is not a style choice that we should enforce.